met-api/src/met_api/services/survey_service.py

```diff
@@ -33,12 +33,25 @@ def supply_banner_url(engagement):
         engagement['banner_url'] = ObjectStorageService.get_url(engagement.get('banner_filename', None))
         return engagement
 
+    @classmethod
+    def get_surveys(cls, unlinked=False):
```

Comment:
I don't get the optional parameter here, isn't it better to just call the method?

Reply:
How do you mean just call the method?

I need to either get all surveys or just the unlinked ones.